corehq/apps/registration/views.py

```diff
@@ -126,15 +126,17 @@ def _create_new_account(self, reg_form):
             web_user.phone_numbers.append(reg_form.cleaned_data['phone_number'])
             web_user.save()
 
-        is_mobile = reg_form.cleaned_data.get('is_mobile')
         email = new_user.email
-
         properties = {}
-        if is_mobile:
-            toggles.MOBILE_SIGNUP_REDIRECT_AB_TEST_CONTROLLER.set(
-                email, True)
-            variation = toggles.MOBILE_SIGNUP_REDIRECT_AB_TEST.enabled(email, toggles.NAMESPACE_USER)
-            properties = {"mobile_signups_test_march2018test": "variation" if variation else "control"}
+
+        if self.request.user_agent.is_mobile:
+            toggles.MOBILE_SIGNUP_REDIRECT_AB_TEST_CONTROLLER.set(email, True)
+            variation = toggles.MOBILE_SIGNUP_REDIRECT_AB_TEST.enabled(
+                email, toggles.NAMESPACE_USER
+            )
+            properties = {
```

Comment:
Probably worth updating this name so the test results are diluted

Reply:
i think the decision was to keep the name the same

Reply:
ok